android/external/downloads/Makefile

```diff
@@ -66,8 +64,13 @@ nettle-$(NETTLE_VERSION)-patched.tar.gz: nettle-$(NETTLE_VERSION).tar.gz
 	rm -rf nettle-$(NETTLE_VERSION)
 
 botan.src.stamp:
-	git clone git@github.com:plancksecurity/botan.git -b $(BOTAN_VERSION)
-	touch $@
+# just check the folder exist for the moment, so we use a submodule.
+	@if [ -d $(BOTAN_SRC) ]; then \
+		touch $@; \
+	else \
+		echo "Error: botan folder does not exist."; \
+		exit 1; \
+	fi
 
 nettle-$(NETTLE_VERSION).tar.gz:
 	wget -nc https://ftp.gnu.org/gnu/nettle/nettle-$(NETTLE_VERSION).tar.gz
```

Comment:
For example, here. No nettle needed at all anymore.

Reply:
We still have the option to use nettle in the jni wrapper. Maybe this is not needed anymore for anything?

Reply:
Not needed, removed nettle.